plugins/recorded_future/help.md

```diff
@@ -2628,6 +2885,7 @@ This plugin does not contain any troubleshooting information.
 
 # Version History
 
+* 5.1.0 - Handle 404 Not Found Error in `Lookup IP`, `Lookup Domain`, `Lookup Hash`, `Lookup Alert`, `Lookup Vulnerability`, `Lookup URL`, `Lookup Malware` actions | Create unit_tests for `Lookup IP`, `Lookup Domain`, `Lookup Hash`, `Lookup Alert`, `Lookup Vulnerability`, `Lookup URL`, `Lookup Malware` actions | Update example outputs in help.md | Update keywords | Update key features
```

Comment:
Please make this PR into a major version bump `6.0.0` because we change the plugin output schema for multiple actions.